### DISCUSSION TITLE
Configuartion data

### DISCUSSION THREAD

**Discussion by froggi1930** [WillCodeForCats/solaredge-modbus-multi#602] [Q&A]
Hi there,

I have successfully set up my inverters and batteries. I wonder if the integration creates some kind of configuartion file in which the modbus id of the components is stored...

**Comment by WillCodeForCats** [comment]
All configuration is available in Home Assistant, there is no need to edit files.

**Reply by froggi1930** [reply]
I understand... is there a way ro find out which modbus id is used for each component. Your integration performs an autodetect, but I need to enter the id´s in a openWB config, but fail to find the id´s.

**Reply by WillCodeForCats** [reply]
The components do not have modbus ID when reading through TCP/IP. The inverter proxies everything.

**Reply by froggi1930** [reply]
... that means if I have multiple inverters and batteries they all get connected through the master?

**Reply by WillCodeForCats** [reply]
Yes, the inverter with the TCP connection proxies the others over the leader/follower chain or other inverter-specific communications. Inverters can be addressed by unit ID, batteries and meters can't. Auto detection for meters and batteries checks registers, not modbus ID.

If you know the inverter IDs (which you had to enter at setup time) then that's all there is to know with the current way modbus for solaredge works. You can find view the inverter ID anytime by clicking "reconfigure" from the device menu in the integration panel.
